presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/NoisySumAvgAggregationUtils.java

```diff
@@ -56,7 +56,7 @@ public static void updateNoisySumState(NoisySumState state, double value, double
         state.setSum(state.getSum() + clippedValue);
     }
 
-    public static void combineNoisySumStates(NoisySumState state, NoisySumState otherState)
+    public static void combineNoisySumStates(NoisySumAvgState state, NoisySumAvgState otherState)
```

Comment:
Should the name of this function be updated? Could just be `combineStates`, since the noisy/sum/avg stuff is implied by the parent class name.